docs/library/select.rst

```diff
@@ -12,21 +12,19 @@ This module provides functions to efficiently wait for events on multiple
 Functions
 ---------
 
-.. function:: poll()
```

Comment:
This is actually a function, and also in CPython it's documented as a function.

Reply:
agree , I will revert this from the PR.

Reply:
although this leaves me puzzled how the language can allow a function and a class with the same name.

Reply:
I'm not sure exactly the reasoning here, but let's just follow CPython.